components/script/dom/webidls/VR.webidl

```diff
@@ -1,11 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
-
-// https://w3c.github.io/webvr/#interface-navigator
-[NoInterfaceObject]
-interface VR {
-  [Pref="dom.webvr.enabled"]
-  Promise<sequence<VRDisplay>> getDisplays();
```

Comment:
It is not clear to me that eliminating WebVR from our implementation is the right choice right now. It's useful to test against the existing WebVR content.

Reply:
I'm not! `navigator.vr` is old WebVR (it's not in the spec anymore). I still support `navigator.getVRDisplays()` which is the new thing.

Reply:
https://immersive-web.github.io/webvr/spec/1.1/#interface-navigator

Reply:
I just found it convenient to repurpose the VR object as XR since it has the same purpose and I didn't want to duplicate the messaging/syncing logic